app/src/main/java/fr/free/nrw/commons/utils/DialogUtil.kt

```diff
@@ -160,19 +160,19 @@ object DialogUtil {
         onNegativeBtnClick: Runnable? = null,
         customView: View? = null,
         cancelable: Boolean = true
-    ) {
+    ): AlertDialog? {
 
         /* If the custom view already has a parent, there is already a dialog showing with the view
          * This happens for on resume - return to avoid creating a second dialog - the first one
          * will still show
          */
         if (customView?.parent != null) {
-            return
+            return null;
         }
 
-        showSafely(activity, AlertDialog.Builder(activity).apply {
+        return showSafely(activity, AlertDialog.Builder(activity).apply {
             title?.also{setTitle(title)}
-            title?.also{setMessage(message)}
+            message?.also{setMessage(message)}
```

Comment:
Unfortunately, I think I wrongly edited this in the previous PR and this should fix it. The previous would result in an empty title not setting a message even if it was provided